flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/CheckpointConfig.java

```diff
@@ -162,7 +162,7 @@ public CheckpointConfig(Configuration configuration) {
 
     /** Disables checkpointing. */
     public void disableCheckpointing() {
-        configuration.removeConfig(ExecutionCheckpointingOptions.CHECKPOINTING_INTERVAL);
+        configuration.removeConfig(CheckpointingOptions.CHECKPOINTING_INTERVAL);
```

Comment:
Some places in `CheckpointConfig` could also be replaced. e.g.  in `configure` method

Reply:
For the methods marked as @Deprecated in `CheckpointConfig`, I have not modified it, just like `ExecutionCheckpointingOptions.CHECKPOINTING_MODE` is still used in CheckpointConfig#getCheckpointingMode.